Cargo.toml

```diff
@@ -36,7 +36,7 @@ azure_storage = {version="0.9", optional=true}
 azure_storage_blobs = {version="0.9", optional=true}
 indicatif = {version="0.17", optional=true}
 native-tls = {version="0.2", features=["vendored"], optional=true}
-reqwest = {version="0.11", optional=true}
+reqwest = {version="0.11", optional=true, features=["stream"]}
```

Comment:
This should solve the problem described in the issue:

> The put feature of AVML requires the stream feature of the reqwest dependency

...however, it also enables the `stream` feature of `reqwest` even when the `put` feature of the `avml` crate isn't enabled.

If you want to enable `reqwest`'s `stream` feature precisely when `avml`'s `put` feature is enabled I believe this should work:

```toml
[features]
put = ["reqwest/stream", ...]
```

See the [dependency features](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features) docs for more.